FitGoal/FitGoal/Singup&Singin/SocialMediaAuthenticator.swift

```diff
@@ -53,4 +69,127 @@ class SocialMediaAuthenticator: NSObject, GIDSignInDelegate {
             completion?(.success(()))
         }
     }
+    
+    //MARK: - Facebook
+    
+    private func facebookSignIn(sender: UIViewController, completion: @escaping SignInCallback) {
+        let permissions = ["public_profile", "email"]
+        LoginManager().logIn(permissions: permissions, from: sender) { (logInResuts, error) in
+            if let error = error {
+                completion(.failure(error))
+            }
+            else {
+                guard let logInResuts = logInResuts else {
```

Comment:
typo in both names, you missed the `l` before the `t`. Also, _login_ is a noun so you don't need to use uppercase for _In_.